core/model.py

```diff
@@ -23,7 +24,7 @@ def __init__(self, model_file=PATH_TO_CKPT, label_file=PATH_TO_LABELS):
         logger.info('Loading model from: {}...'.format(model_file))
         detection_graph = tf.Graph()
         graph = tf.Graph()
-        sess = tf.Session(graph=detection_graph)
+        _ = tf.Session(graph=detection_graph)
```

Comment:
This is strange. The session should be closed somewhere in the code. This can be done either via `close()` or context manager